packages/devtools_app/lib/src/network/flutter/network_screen.dart

```diff
@@ -186,164 +186,169 @@ class NetworkScreenBodyState extends State<NetworkScreenBody> {
     super.initState();
   }
 
+  Row _buildHttpProfilerControlRow(bool recording) {
+    const double minIncludeTextWidth = 600;
+    return Row(
+      children: [
+        recordButton(
+          recording: recording,
+          minIncludeTextWidth: minIncludeTextWidth,
+          onPressed: () => networkController.startRecording(),
+        ),
+        pauseButton(
+          paused: !recording,
+          minIncludeTextWidth: minIncludeTextWidth,
+          onPressed: () => networkController.pauseRecording(),
+        ),
+        OutlineButton(
+          onPressed: () => networkController.refreshRequests(),
+          child: Label(
+            FlutterIcons.refresh,
+            'Refresh',
+            minIncludeTextWidth: 900,
+          ),
+        ),
+        const SizedBox(width: 8.0),
+        clearButton(
+          onPressed: () {
+            dataTableSource.clearSelection();
+            networkController.clear();
+          },
+        ),
+        const Spacer()
```

Comment:
is this widget needed? What is it "spacing"?

Reply:
Apparently not. Guess I was using it for something else before... Will remove.